data/campaigns/Heir_To_The_Throne/scenarios/01_The_Elves_Besieged.cfg

```diff
@@ -585,6 +585,9 @@
         # httt achievement one
         {IF_VAR besieged_enemies_killed equals 2 (
             [then]
+                [sound]
+                    name="fanfare-short.wav"
+                [/sound]
```

Comment:
So, this is going to play even if you already had the achievement, right? Is that what you want>

If we want a sound to be associated with an achievement, I think it needs to be done _inside_ the `[set_achievement]` tag.

Reply:
built-in sound cue would be nice. Do we list it as a feature request?

Reply:
Sure? Might also be nice to have a preference to disable it…

Reply:
Achievements now have the `sound` attribute which can do this.

Reply:
But if we want, it should be easy to add that now that the sound is played from C++.

Reply:
Actually, it looks like that's already handled in the `sound::play_sound()` method. So it gets enabled/disabled at the same time as other sounds.

Reply:
I meant if we wanted a preference to disable _only_ achievement sounds. Of course, if you've turned _all_ sounds off, then no sounds will play no matter how much the code requests them.

Reply:
That seems unnecessarily specific.